config/exchanges.json

```diff
@@ -2434,76 +2434,6 @@
     },
```

Comment:
I think this case with position of exchange arrows is worth fixing now:


1. New York -> New England

<img width="151" alt="Screenshot 2021-08-24 at 09 47 23" src="https://user-images.githubusercontent.com/3296643/130578216-b0db0a04-6572-4441-a96d-cc17d4380979.png">

The nicely placed exchange in the middle is the old one that is removed, so would be great if the new one could take it's place instead of being far down at the bottom.

_Old exchange_
<img width="594" alt="Screenshot 2021-08-24 at 09 51 15" src="https://user-images.githubusercontent.com/3296643/130578341-80333cc4-cf71-4ad9-ba22-7566ece83200.png">

_New exchange_
<img width="815" alt="Screenshot 2021-08-24 at 09 51 21" src="https://user-images.githubusercontent.com/3296643/130578338-d7c3e37e-d9c0-408f-aa1f-d6052cbfbad6.png">

Reply:
Do you know what the best workflow for updating these? 🤔

Reply:
Copy-paste position and rotation? 😅 Don't think it's scripted

Reply:
#sad... I'll try. :D

Reply:
Should be fixed now. Much nicer.

Before (prod): 
<img width="1552" alt="Screenshot 2021-08-24 at 11 36 49" src="https://user-images.githubusercontent.com/1260305/130594193-9b2d5067-e22c-498b-a541-20dc54baf4e2.png">

After:
<img width="1552" alt="Screenshot 2021-08-24 at 11 36 51" src="https://user-images.githubusercontent.com/1260305/130594213-1555da0f-a41d-4f4f-a45a-9c5042e94f2e.png">